packages/forms/frontend/sirius-components-widget-reference/src/ReferencePropertySection.tsx

```diff
@@ -36,6 +37,13 @@ import {
   GQLSuccessPayload,
 } from './ReferenceWidgetFragment.types';
 
+const useStyles = makeStyles((theme) => ({
```

Comment:
`theme` is not used so it could be replaced by '_' or by nothing.

Reply:
done